From 5ef6fe35411cb219c17c9137726f6a914c928d0e Mon Sep 17 00:00:00 2001
From: Hari Nagalla <hnagalla@ti.com>
Date: Tue, 11 Feb 2025 14:25:27 -0600
Subject: [PATCH 59/68] FROMLIST: arm64: dts: ti: k3-am62a7-sk: Reserve
 main_rti4 for C7x DSP

The main rti4 watchdog timer is used by the C7x DSP, so reserve the
timer in the linux device tree.

Link: https://lore.kernel.org/linux-devicetree/20250206235200.3128163-1-jm@ti.com/
Signed-off-by: Hari Nagalla <hnagalla@ti.com>
Signed-off-by: Judith Mendez <jm@ti.com>
---
 arch/arm64/boot/dts/ti/k3-am62a7-sk.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts b/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts
index c6764a62e063..93cb3d378c31 100644
--- a/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts
+++ b/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts
@@ -826,3 +826,8 @@ &c7x_0 {
 			<&c7x_0_memory_region>;
 	status = "okay";
 };
+
+/* main_rti4 is used by C7x DSP */
+&main_rti4 {
+	status = "reserved";
+};
-- 
2.47.2

